core/src/actions/session.ts

```diff
@@ -13,7 +13,6 @@ export class SessionCreate extends CLSAction {
       email: { required: true },
       password: { required: true },
     };
-    this.permission = { topic: "session", mode: "write" };
```

Comment:
This type shouldn't have existed - this action is not authenticated.

Reply:
It's possible this was here because it had a `write` as a side effect and the `write` was needed for SQLite reasons.

Reply:
At the very least, you'll need to override `isWriteTransaction` to be true then.

Reply:
Ah ha! Fixed up in https://github.com/grouparoo/grouparoo/pull/2207